internal/api/shared/crypto/jwt.go

```diff
@@ -93,6 +102,21 @@ func (j *jwtImpl) CreateJWTCookies(token string) (*http.Cookie, *http.Cookie) {
 	return headerPayloadCookie, signatureCookie
 }
 
+func (j *jwtImpl) Parse(token string) (*JWTCustomClaims, error) {
```

Comment:
hopefully using the  `JWTMiddleware` you shouldn't need to do that. It should have already extracted / converted the token for you.

Reply:
```go
user := c.Get("user").(*jwt.Token)
claims := user.Claims.(crypto.JWTCustomClaims)
```

https://github.com/labstack/echo-jwt